offices/_single.html

```diff
@@ -106,23 +110,21 @@ <h2 class="h3">{{ resource.title | safe }}</h2>
     </section>
     {% endif %}
 
-    {% if vars.sub_pages %}
+    {% for page in vars.sub_pages if page.show_in_office == "on" %}
+    {% if loop.first %}
```

Comment:
That works if we don't have show_in_office logic. For example, we have lots of cases where many sub-pages exist and we don't want to show any of them yet and the section shows up.